src/flowgraph/resampler/README.md

```diff
@@ -20,7 +20,7 @@ Only do this once, when you open your stream. Then use the sample resampler to p
             2, // channel count
             44100, // input sampleRate
             48000, // output sampleRate
-            MultiChannelResampler::Medium); // conversion quality
+            MultiChannelResampler::Quality::Medium); // conversion quality
```

Comment:
Thanks. Note that you can make fixes to *.md files right on GitHub using their markdown editor.

Reply:
Ack.